### PR TITLE
Update kernel-command-line-config.adoc

### LINKED ISSUE #2083: cmdline.txt sytax (dwc_otg.lpm_enable)
The documentation re. Command Line Options (https://www.raspberrypi.org/documentation/computers/configuration.html#command-line-options) says "dwc_otg.lpm_enable: turns off Link Power Management ...".  It seemed incredibly counter-intuitive that an option called "enable" would turn something off until I noticed that the actual syntax in one of my cmdline.txt files was "dwc_otg.lpm_enable=0".  It would seem appropriate to define the acceptable values for this option (and others?) in the documentation.

### PR DESCRIPTION
**Pull request by andrum99**
Fixes #2083

### CODE REVIEW

**Comment by andrum99** [comment]
I've added notes to the two DWC OTG settings which explain that they don't apply to the main USB controller on Pi 4.

**Review by aallan — APPROVED** [review]

**Comment by PeterPablo** [comment]
The sentence of line 76 is almost identical to line 73.

File: documentation/asciidoc/computers/configuration/kernel-command-line-config.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ The Linux kernel accepts a command line of parameters during boot. On the Raspbe
 sudo nano /boot/cmdline.txt
 ----
 
-Note that we have to use `sudo` to edit anything in the boot partition, and that all parameters in `cmdline.txt` must be on the same line (no carriage returns).
+NOTE: We have to use `sudo` to edit anything in the boot partition, and that all parameters in `cmdline.txt` must be on the same line (no carriage returns).
```
Comment (lurch) [inline documentation/asciidoc/computers/configuration/kernel-command-line-config.adoc:9]: Should the "that" be removed?

Comment (aallan) [reply documentation/asciidoc/computers/configuration/kernel-command-line-config.adoc:9]: Oh the second "that", yes it should. Fixed in a191a74.

**Pull request merged**